templates/ncrc5-intel-classic.mk

```diff
@@ -77,6 +79,14 @@ $(error Options DEBUG and TEST cannot be used together)
 endif
 endif
 
+
+ifdef USE_R4
+REAL_PRECISION := -r4
+CPPDEFS += -DOVERLOAD_R4
+else
+REAL_PRECISION := -r8
+endif
+
```

Comment:
@bensonr 
should we be using `-real-size 32` and `real-size 64` instead?

Reply:
@uramirez8707 - Yes, I think we should be using the updated `-real-size` for both OneAPI and classic ifort.